package.json

```diff
@@ -81,7 +81,8 @@
                         "bat": "",
                         "shellscript": "bash",
                         "fsharp": "fsi",
-                        "csharp": "scriptcs"
+                        "csharp": "scriptcs",
+                        "vbscript": "cscript //Nologo"
```

Comment:
Agree. It would be better to distinguish vb and vbs in VS Code. Since VS Code does not distinguish vb and vbs itself, So I suggest adding executorMapByFileExtension config as I said before. Are you willing to update your PR to implement this? Or, I am OK to merge the current PR and I could add code logic to implement executorMapByFileExtension based on that.